File: Java/DailyFlash/August/21August/Program4.java


class Pattern {

	public static void main(String[] args) {
		
		int num=1;
		for(int i=1;i<=16;i++) {

			if(i%4==0) {

				System.out.println(2*num);
				num=1;
			}
			else {
				System.out.print(2*num+" ");
				num++;
			}
		}
	}
}

File: PPA/DailyFlash/July/30July/Program4.c


#include<stdio.h>

void main() {

	for(int i=1; i<=4; i++) {
		int num=1;
		for(int j=3;j>=i;j--) {

			printf("  ");
			num++;
		}
		for(int k=1;k<=i*2-1;k++) {

			printf("%d ",num);
			num++;
		}
		printf("\n");
	}
}
